ui/src/app/views/cohort-builder/search/wizard-criteria-group/wizard-criteria-group.component.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Component, OnDestroy, Output, EventEmitter, OnInit, ChangeDetectorRef,
+  ViewEncapsulation, Input, ViewChild
+} from '@angular/core';
+import { BroadcastService } from '../service';
+import { SearchCriteria } from '../model';
+import { Subscription } from 'rxjs/Subscription';
+
+@Component({
+  selector: 'app-wizard-criteria-group',
+  templateUrl: 'wizard-criteria-group.component.html',
+  styleUrls: ['wizard-criteria-group.component.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class WizardCriteriaGroupComponent implements OnInit, OnDestroy {
+
+  criteriaList: SearchCriteria[] = [];
+  criteriaType: string;
+  @ViewChild('groupDiv') groupDiv: any;
+  private criteriaSubscription: Subscription;
+  private criteriaTypeSubscription: Subscription;
+
+  constructor(private changeDetectorRef: ChangeDetectorRef,
+              private broadcastService: BroadcastService) {}
+
+  ngOnInit(): void {
+    this.criteriaSubscription = this.broadcastService.selectedCriteria$
+      .subscribe(criteria => {
+        this.updateCriteriaList(criteria);
+        this.updateCriteriaGroupView();
+      });
+    this.criteriaTypeSubscription = this.broadcastService.selectedCriteriaType$
+      .subscribe(criteriaType =>
+        this.criteriaType = criteriaType);
+  }
+
+  public getTypeDescription(): string {
+    if (this.criteriaType === 'icd9'
+    || this.criteriaType === 'icd10'
```

Comment:
Might be good to indent this 4 spaces to clarify to the reader that it's a continuation of the if statement.